### PR TITLE
Add link to CUE Activation Keys shop

### PR DESCRIPTION
**Pull request by abhigyanghosh30**
## Done

- Reopening the keys shop when the CUE Activation Keys go live in production.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Go to /credentials
    - There will be two buttons for individual exams and keys
- Change the CONTRACTS_API_URL to production in `.env.local`
    - Restart dotrun
    - Go to /credentials again
    - There should be a text in italics as things are not in production yet 

## Issue / Card

Fixes [#WD-5840](https://warthogs.atlassian.net/browse/WD-5840)

### In Staging

image
image

### In Produciton
image
image

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: static/js/src/advantage/credentials/components/CredKeyShop/CredKeyShop.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { currencyFormatter } from "advantage/react/utils";
 const CredKeyShop = () => {
   const CUEExamKey = {
     id: "cue-activation-key",
-    longId: "lAMGrt4buzUR0-faJqg-Ot6dgNLn7ubIpWiyDgOrsDCg",
+    longId: "lAEPoNKYCFgZGmddQXplCtcnf5wMpYUXloGygjPK3y0E",
```
[the remark this answers is not in the document]
Comment (jpmartinspt) [reply static/js/src/advantage/credentials/components/CredKeyShop/CredKeyShop.tsx:15]: There is no guarantee they will be the same in prod, we need to account for the change. Maybe get the listings from the API and return the listing where ID == "cue-activation-key" to the frontend.

File: webapp/shop/cred/views.py [hunk 2]
```diff
@@ -708,3 +714,42 @@ def issue_badges(trueability_api, credly_api, **kwargs):
             return (flask.jsonify(new_badge), 500)
     # 403 Forbidden. Request was valid but the server is refusing action
     return flask.jsonify({"status": "badge_not_issued"}), 403
+
+
+@shop_decorator(area="cred", permission="user", response="json")
```
Comment (jpmartinspt) [inline webapp/shop/cred/views.py:719]: Can we make this one endpoint that returns all by default, but allows you to send a `type="keys|exams"` param if you want only one or the other?

Comment (abhigyanghosh30) [reply webapp/shop/cred/views.py:719]: Done

File: static/js/src/advantage/credentials/components/CredExamShop/CredExamShop.tsx [hunk 3]
```diff
@@ -88,6 +74,24 @@ const CredExamShop = () => {
...
+    }
+    for (const examDescription of ExamProductDescriptions) {
+      for (const exam of ExamData) {
+        if (exam.id === examDescription.id) {
+          Object.assign(examDescription, exam);
+        }
+      }
+    }
+    setExamProducts(ExamProductDescriptions);
+    console.log(ExamProductDescriptions);
```
Comment (jpmartinspt) [inline static/js/src/advantage/credentials/components/CredExamShop/CredExamShop.tsx:89]: Remove log

**Review by jpmartinspt — APPROVED** [review]

**Pull request merged**